crates/ruff_linter/src/rules/pydoclint/rules/check_docstring.rs

```diff
@@ -121,15 +122,15 @@ impl Violation for DocstringExtraneousReturns {
 }
 
 /// ## What it does
-/// Checks for functions with yield statements missing a "yields" section in
-/// their docstring.
+/// Checks for functions with `yield` statements that do not have "Yields" sections in
+/// their docstrings.
```

Comment:
I like the wording here better than the one for returns above. It's simpler (less plural form). Maybe we can align the two?